src/services/filters.js

```diff
@@ -21,11 +22,164 @@ goog.require('ngeo');
+ *      <!-- will Become 600'000, 200'001 -->
+ *      <p>{{[2600000.1234, 1200000.5678] | ngeoSwissCoordinates}}</p>
+ *      <!-- will Become 2'600'000, 1'200'001 -->
+ *
+ * Example with defined prefix, separator and suffix (you can
+ *     not use the colon symbole).
+ *
+ *      <p>{{[600000.1234, 200000.5678] | ngeoSwissCoordinates:[:; :]}}</p>
+ *      <!-- will Become [600'000; 200'001] -->
+ *      <p>{{[2600000.1234, 1200000.5678] | ngeoSwissCoordinates:[:; :]}}</p>
+ *      <!-- will Become [2'600'000; 1'200'001] -->
+ *
+ * @return {function(ol.Coordinate, string, string, string): string} A
+ *     function to format numbers into an Swiss coordinates string.
+ * @ngInject
+ * @ngdoc filter
+ * @ngname ngeoSwissCoordinates
+ */
+ngeo.SwissCoordinates = function() {
+  /**
+   * @param {ol.Coordinate} coordinates Array of two numbers.
+   * @param {string=} opt_prefix Optional prefix.
+   * @param {string=} opt_separator Optional separatator. default to ' '.
+   * @param {string=} opt_suffix Optional suffix.
+   * @return {string} Swiss formated coordinates.
+   */
+  var filterFn = function(coordinates, opt_prefix, opt_separator, opt_suffix) {
+    var prefix = opt_prefix ? opt_prefix : '';
+    var separator = opt_separator ? opt_separator : ', ';
+    var suffix = opt_suffix ? opt_suffix : '';
+    var x = Math.round(coordinates[0]);
+    x = x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, '\'');
+    var y = Math.round(coordinates[1]);
+    y = y.toString().replace(/\B(?=(\d{3})+(?!\d))/g, '\'');
+    return prefix + x + separator + y + suffix;
+  };
+  return filterFn;
+};
+
+ngeo.module.filter('ngeoSwissCoordinates', ngeo.SwissCoordinates);
+
+
+/**
+ * Format a couple of numbers as a East North coordinates.
+ *
+ * Example without parameters:
+ *
+ *      <p>{{[7.1234, 46.9876] | ngeoEastNorthCoordinates}}</p>
+ *      <!-- will Become 7 E 47 N -->
+ *
+ * Example with defined fractionDigits, prefix, separator and suffix (you can
+ *     not use the colon symbole).
+ *
+ *      <p>{{[7.1234, 46.9876] | ngeoEastNorthCoordinates:2:[:; :]}}</p>
+ *      <!-- will Become [7.12 E; 46.99 N] -->
+ *
+ * @return {function(ol.Coordinate, number, string, string, string): string} A
+ *     function to format numbers into an East North coordinates string.
+ * @ngInject
+ * @ngdoc filter
+ * @ngname ngeoEastNorthCoordinates
+ */
+ngeo.EastNorthCoordinates = function() {
+  /**
+   * @param {ol.Coordinate} coordinates Array of two numbers.
+   * @param {number=} opt_fractionDigits number of digit, Same as
+   *     {@link ol.coordinate.format} opt_fractionDigits parameters.
+   * @param {string=} opt_prefix Optional prefix.
+   * @param {string=} opt_separator Optional separatator. default to ' '.
+   * @param {string=} opt_suffix Optional suffix.
+   * @return {string} East north formated coordinates.
+   */
+  var filterFn = function(coordinates, opt_fractionDigits, opt_prefix,
+      opt_separator, opt_suffix) {
```

Comment:
Instance og havine a `opt_prefix`, `opt_separator`, `opt_suffix` can't we have a template and used for example https://www.npmjs.com/package/es6-template-strings